types/office-js-preview/index.d.ts

```diff
@@ -60513,6 +60978,13 @@ declare namespace Excel {
         }
         /** An interface for updating data on the ChartSeriesFormat object, for use in `chartSeriesFormat.set({ ... })`. */
         interface ChartSeriesFormatUpdateData {
+            /**
+            * Represents the fill format of a chart series, which includes background formatting information.
+            *
+            * @remarks
+            * [Api set: ExcelApi 1.1]
+            */
+            fill?: Excel.Interfaces.ChartFillUpdateData;
```

Comment:
@wandyezj There are also a bunch of APIs related to ChartFill added in this update, but they have API set tags other than beta (many are ExcelApi 1.1).